src/generated/prisma-client/prisma-schema.js

```diff
@@ -373,6 +543,11 @@ input UserCreateInput {
   votes: VoteCreateManyWithoutUserInput
 }
 
+input UserCreateOneInput {
```

Comment:
@pantharshit00 I understand about the generate and all the stuff related to images, but why this news Inputs appear here?

Thanks for your help :)